src/components/message/ErrorReport.tsx

```diff
@@ -166,9 +159,16 @@ export const ErrorReport = ({ components }: IErrorReportProps) => {
                       onClick={handleErrorClick(error)}
                       onKeyDown={handleErrorClick(error)}
                     >
-                      {getParsedLanguageFromText(error.message, {
-                        disallowedTags: ['a'],
-                      })}
+                      {error.message.includes(AsciiUnitSeparator)
+                        ? getParsedLanguageFromText(
+                            error.message.substring(error.message.indexOf(AsciiUnitSeparator) + 1),
+                            {
+                              disallowedTags: ['a'],
+                            },
+                          )
+                        : getParsedLanguageFromText(error.message, {
+                            disallowedTags: ['a'],
+                          })}
```

Comment:
Maybe you could separate out this logic into a function that takes in error.message and returns the substring if the separator-symbol is present and otherwise just returns the string as is? I think the code would look cleaner.